orc8r/lib/go/service/config/service_config.go

```diff
@@ -33,6 +33,7 @@ var (
 	configDir         = "/etc/magma/configs"
 	oldConfigDir      = "/etc/magma"
 	configOverrideDir = "/var/opt/magma/configs/"
+	specDir           = "/etc/magma/configs/orc8r/swagger_specs/"
```

Comment:
Nit for uniformity: I know we don't follow this necessarily, but my preference is to *not* include the trailing slash for directory names. Doesn't really matter one way or the other, just ideally we'd be doing it always the same way